ui/README.md

```diff
@@ -3,58 +3,69 @@ Dashboard made with [elm-bootstrap](http://elm-bootstrap.info/).
 
 ## Install
 
-### Install GUI as a part of Mainflux build
+### Docker container GUI build
 
-Install Elm (https://guide.elm-lang.org/install.html) and then
-
-```
-git clone https://github.com/mainflux/mainflux
-cd mainflux/ui
-make
-```
-
-This will produce `index.html` in the _ui_ directory. In order to use it, `cd`
-to _ui_ and do
+Install Docker (https://docs.docker.com/install/) and Docker compose
+(https://docs.docker.com/compose/install/), `cd` to Mainflux root directory and
+then
 
-`make run`
+`docker-compose -f docker/docker-compose.yml up`
 
-### Build a standalone native GUI
+if you want to launch a whole Mainflux docker composition or just
 
-Install Elm (https://guide.elm-lang.org/install.html), `cd` to _ui_ and then
+`docker-compose -f docker/docker-compose.yml up ui`
 
-`elm make --optimize src/Main.elm`
+if you want to launch just GUI.
 
-This will produce `index.html` in the _ui_ directory. In order to use it do
 
-`make run`
+### Native GUI build
 
-### About Elm `make`
+Install Elm (https://guide.elm-lang.org/install.html) and then edit
+*src/Env.elm* by replacing
 
-`make` does `elm make src/Main.elm`.
+```
+env =
+    { protocol = ""
```

Comment:
There should be some default values IMHO.

Reply:
No default value needed, cause we need it to work out of the box in the docker composition. We only need to input some value if it's natively deployed or user wants a special setting.

Reply:
Why can not it work natively with default value, and be controlled in the Docker via ENV variable?